### DISCUSSION TITLE
Discord Provider Not Working

### DISCUSSION THREAD

**Discussion by masnwilliams** [nextauthjs/next-auth#3798] [Help]
Hello! I am trying to get the Discord login provider working but it doesn't seem to want to work. I have Google and GitHub functioning but can't get Discord working.

I currently have `http://localhost:3000/api/auth/callback/discord` as my Redirect in the Discord OAuth2. I am not sure what I am doing wrong.

Any help is appreciated.

**Comment by ryanbarr** [comment]
Hey @johnmasonwilliams,

Would you be able to provide some insight into what you're experiencing? When you say that the provider is not working, what symptoms and errors are you seeing?

**Reply by masnwilliams** [reply]
I got it working. Thanks for the response though!

**Comment by pablojsx** [comment]
Having a similar issue, it only works with localhost but not with my domain name.
